controllers/topics/index.js

```diff
@@ -23,7 +23,8 @@ const {
   UserTopicHistory,
   sequelize,
   User,
-  TopicInvitations
+  TopicInvitations,
+  CommunityMessageFormat
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `CommunityMessageFormat` is added correctly.
```diff
+  CommunityMessageFormat
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->